eras/conway/test-suite/cddl-files/conway.cddl

```diff
@@ -117,11 +112,12 @@ governance_action =
   // treasury_withdrawals_action
   // no_confidence
   // new_committee
+  // new_constitution
   ]
 
 parameter_change_action = (0, protocol_param_update)
 
-hard_fork_initiation_action = (1, protocol_version)
+hard_fork_initiation_action = (1, [protocol_version])
```

Comment:
why did we put this in an array?

Reply:
Because that is the proper format for the protocol version (See cddl for pparamsupdate for example)

IMHO. `protocol_version` should be defined as an array instead, but that's a much bigger change:
```
protocol_version = [major_protocol_version, uint]
```

Reply:
oh, I was confused, indeed I thought that `protocol_version = [major_protocol_version, uint]`. but it is `(...)`.